src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -480,11 +480,13 @@ class ExposureSubmissionCoordinator: NSObject, RequiresAppDependencies {
 
 	private func showQRScreen(testInformation: CoronaTestQRCodeInformation?, isLoading: @escaping (Bool) -> Void) {
 		let testInformationSuccess: (CoronaTestQRCodeInformation) -> Void = { [weak self] testQRCodeInformation in
-			if let oldTest = self?.model.coronaTestService.coronaTest(ofType: testQRCodeInformation.testType),
-			   oldTest.testResult != .expired && oldTest.testResult != .invalid {
-				self?.showOverrideTestNotice(testQRCodeInformation: testQRCodeInformation, submissionConsentGiven: true)
+			guard let self = self else { return }
+			if let oldTest = self.model.coronaTestService.coronaTest(ofType: testQRCodeInformation.testType),
+			   oldTest.testResult != .expired,
+			   (oldTest.type == .antigen && !self.model.coronaTestService.antigenTestIsOutdated) {
```

Comment:
```suggestion
			   (oldTest.type != .antigen || !self.model.coronaTestService.antigenTestIsOutdated) {
```

I we need to check like that, otherwise it would never be shown for pcr tests

Reply:
Or, maybe even a bit easier to read:

```suggestion
			   !(oldTest.type == .antigen && self.model.coronaTestService.antigenTestIsOutdated) {
```